api/test/docker/apisix_config.yaml

```diff
@@ -42,6 +42,7 @@ apisix:
 
 nginx_config:
   error_log_level: "debug"
+  worker_processes: "1"
```

Comment:
need a comment, only for debug now

Reply:
Only for debug? Maybe I will add it to master if it can make the e2e test passed.

Reply:
we need to find the real bug. 

I think we should remove this field in E2E case.